presidio-anonymizer/presidio_anonymizer/engine/anonymizers/hash.py

```diff
@@ -0,0 +1,13 @@
+import hashlib
+
+# TODO implement + test
+from anonymizers.anonymizer import AnonymizerAbstract
+
+
+class Hash(AnonymizerAbstract):
+    def __init__(self,
+                 old_text: str):
+        self.text = old_text
+
+    def annonymize(self):
+        return hashlib.sha256(self.old_text.hash_str.encode()).hexdigest()
```

Comment:
I wonder why is this implementation here? Beyond scope of the user story.
I am pretty sure it belongs to user story https://dev.azure.com/csedevil/Presidio-internal/_workitems/edit/2544/
This helps to reduce the size of your PR

Reply:
It is beyond the scope. I can remove it if it's that critical.